src/pages/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { tabs } from '../../data/tabs';
+
+export const TabsPage = () => {
+  const { tabId } = useParams();
+  const isCorrectTab = tabs.some(tab => tab.id === tabId);
+  const tabContent = tabs.find(tab => tab.id === tabId);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={classNames({
+                'is-active': tabId === tab.id,
+              })}
+            >
+              <Link to={`../${tab.id}`}>{tab.title}</Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {isCorrectTab && tabId
```

Comment:
The 'isCorrectTab' in the ternary operator is redundant. You can just check if 'tabContent' exists, since 'tabContent' will be undefined if 'tabId' does not match any tab id.